Packs/Sigma/Scripts/SigmaConverttoQuery/SigmaConverttoQuery.py

```diff
@@ -0,0 +1,96 @@
+
+from sigma import exceptions
+from sigma.backends.carbonblack import CarbonBlackBackend
+from sigma.backends.cortexxdr import CortexXDRBackend
+from sigma.backends.elasticsearch import LuceneBackend
+from sigma.backends.microsoft365defender import Microsoft365DefenderBackend
+from sigma.backends.qradar import QradarBackend
+from sigma.backends.sentinelone import SentinelOneBackend
+from sigma.backends.splunk import SplunkBackend
+from sigma.rule import SigmaRule
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+SIEMS = {
+    "xql": CortexXDRBackend(),
+    "splunk": SplunkBackend(),
+    "sentinel_one": SentinelOneBackend(),
+    "qradar": QradarBackend(),
+    "microsoft_defender": Microsoft365DefenderBackend(),
+    "carbon_black": CarbonBlackBackend(),
+    "elastic": LuceneBackend()
+}
+
+
+def get_sigma_dictionary(indicator: str) -> str:
+    """
+    Find the Sigma rule dictionary for a given indicator value.
+
+    Args:
+        indicator (str): The indicator value to search for.
+
+    Returns:
+        dict: The Sigma rule dictionary.
+
+    Raises:
+        DemistoException: If the indicator is not found or Sigma dictionary cannot be loaded.
+    """
+
+    demisto.debug(f'Starting search for indicator: {indicator}')
+    search_result = demisto.executeCommand('findIndicators', {'value': indicator})
+
+    if isError(search_result):
+        return_error(f'Failed to find indicator {indicator}. Error: {get_error(search_result)}')
+
+    indicators = search_result[0]['Contents']
```

Comment:
This can be streamlined with `execute_command()`.

Reply:
Done